docker/build-tools/Dockerfile

```diff
@@ -116,15 +115,30 @@ ADD https://github.com/cli/cli/releases/download/v${GH_VERSION}/gh_${GH_VERSION}
 RUN dpkg -i /tmp/gh_${GH_VERSION}_linux_${TARGETARCH}.deb
 RUN mv /usr/bin/gh ${OUTDIR}/usr/bin
 
+# Install protoc-gen-validate
+RUN set -eux; \
```

Comment:
go get no longer works as [the](https://github.com/bufbuild/protoc-gen-validate/blob/4e1870c281a44e47ae213a5f8bd704b634b0d693/go.mod#L1) references `github.com/envoyproxy/protoc-gen-validate`